apps/ima/index.js

```diff
@@ -33,9 +33,38 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/exception',
```

Comment:
can you keep the comment that this is to be updated as steps are added

Reply:
Added the comment to this step.